samples/cpp/image_generation/README.md

```diff
@@ -24,6 +24,12 @@ Users can change the sample code and play with the following generation paramete
 > [!NOTE]
 > Image generated with HuggingFace / Optimum Intel is not the same generated by this C++ sample: C++ random generation with MT19937 results differ from `numpy.random.randn()` and `diffusers.utils.randn_tensor` (uses `torch.Generator` inside). So, it's expected that image generated by Diffusers and C++ versions provide different images, because latent images are initialize differently.
 
+## OpenVINO blogs about Image Generation with GenAI 
+
+Please follow the following blogs to setup your first hands-on experience with samples.
```

Comment:
we already have `
Follow [Get Started with Samples](https://docs.openvino.ai/2024/learn-openvino/openvino-samples/get-started-demos.html) to run the sample` below

Reply:
@ilya-lavrenov, but https://docs.openvino.ai/2024/learn-openvino/openvino-samples/get-started-demos.html isn't related to OpenVINO GenAI scenarios, right?

Reply:
OpenVINO and OpenVINO GenAI samples have the same build instructions.
All other steps how to run samples are located within this README.md.

I don't understand why should we refer to some blogs, while we have our own documentation.
The goal of blogs is to advertise OpenVINO; while referring to blogs from our docs, we advertise medium.com.